src/charm.py

```diff
@@ -420,7 +420,7 @@ def _on_pebble_ready(self, event) -> None:
             self.unit.set_workload_version(version)
         else:
             logger.debug(
-                "Cannot set workload version at this time: could not get Alertmanager version."
+                "Cannot set workload version at this time: could not get Prometheus version."
             )
 
     def _update_config(self, container) -> bool:
```

Comment:
Seems like `_update_config` is not used anywhere. This is odd.

Reply:
Let's follow this issue here: https://github.com/canonical/prometheus-k8s-operator/issues/399